cmd/version.go

```diff
@@ -5,20 +5,32 @@ package cmd
 
 import (
 	"fmt"
+	"runtime/debug"
 
 	"github.com/spf13/cobra"
 )
 
 // GoBlueprintVersion is the version of the cli to be overwritten by goreleaser in the CI run with the version of the release in github
 var GoBlueprintVersion string
 
+func getGoBlueprintVersion() string {
+	if len(GoBlueprintVersion) != 0 {
+		return GoBlueprintVersion
+	}
+	if info, ok := debug.ReadBuildInfo(); ok {
+		return info.Main.Version
+	}
+	return "unknown"
```

Comment:
Nit: Maybe we can return something else rather than "unknown" since that is a big ambiguous

Reply:
true, will try to come up with something better